service/src/main/java/bio/terra/pipelines/common/utils/PipelineInputTypesEnum.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.pipelines.common.utils;
+
+import bio.terra.common.exception.ValidationException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public enum PipelineInputTypesEnum {
+  STRING("string") {
+    @Override
+    public String cast(String fieldName, Object value) {
+      String stringValue = objectMapper.convertValue(value, String.class);
+      if (stringValue.isBlank()) {
```

Comment:
note this does throw an error if `stringValue` is null, so we're counting on nulls having been validated before this

Reply:
(now we explicitly validate not null right above)